pubmedcentral/download_and_convert_to_md.py

```diff
@@ -0,0 +1,109 @@
+import argparse
+import functools
+import multiprocessing as mp
+import os
+import tarfile
+
+from tqdm import tqdm
+
+cur_dir = os.getcwd()
+
+parser = argparse.ArgumentParser(description="Convert xml documents to markdown.")
+parser.add_argument("--filelist", help="The path to the filelist.txt file.")
+parser.add_argument(
+    "--output_dir", default="data/raw/", help="Where the markdown files go."
+)
+parser.add_argument("--total_docs", default=0, type=int, help="Total number of documents to convert, for debugging.")
+
+BASE_URL = "https://ftp.ncbi.nlm.nih.gov/pub/pmc/"
+
+
+def download(f_url: str, output_dir: str):
```

Comment:
Is there a reason to do a subprocess call to `wget` over making the request from python? If something like the code from https://github.com/r-three/licensed-pile/blob/d2dadf0de22c832459be2fc0e11bc2387b625e34/wikiscrape/utils.py#L20 will work for you we can move that into a shared location for reuse.

Can you test if that snippet works for you?

Reply:
That works for me, let's get that added to a shared location on the main branch!

Reply:
@blester125 Pinging you about getting your get_page function pushed to the main branch. I'd be happy to create a new pull request just for getting that added. Maybe we can create a utils folder that's shared across domains?